OHRM5X-2395: AWS CLI Addition

devishke-orange opened the pull request.

devishke-orange committed:
move existing files to docker directory

devishke-orange committed:
added aws code

devishke-orange committed:
added github workflows

devishke-orange committed:
fixed github workflow

devishke-orange committed:
Add GPLv3 License

devishke-orange committed:
added GPLv3 copyright header and changed update URL

devishke-orange committed:
updated README install instructions

RajithaKumara commented on aws/README.md:
```diff
@@ -0,0 +1,72 @@
+
+## User Guide
+
+Our user guide for setting up your EC2 instance with the OrangeHRM AMI can be found [here](https://docs.google.com/document/d/1E4mSuN21M54IHjWaXx9DNE1SEtm9EEvj9QWffeVHbm4/edit?usp=sharing). This README will only provide list the available commands.
```
change to help portal link

devishke-orange replied:
changed

devishke-orange commented on aws/README.md:
```diff
@@ -0,0 +1,72 @@
+
+## User Guide
+
+Our user guide for setting up your EC2 instance with the OrangeHRM AMI can be found [here](https://docs.google.com/document/d/1E4mSuN21M54IHjWaXx9DNE1SEtm9EEvj9QWffeVHbm4/edit?usp=sharing). This README will only provide list the available commands.
```
Change to help portal link

devishke-orange replied:
changed

RajithaKumara commented on aws/compose.yml:
```diff
@@ -0,0 +1,50 @@
+services:
+  orangehrm:
+    image: orangehrm/orangehrm:5.5
```
```suggestion
    image: orangehrm/orangehrm
```
`latest`

devishke-orange replied:
did not change this since version checking happens from here

RajithaKumara commented on aws/assets/license.txt:
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 2, June 1991
```
we may have to change the license for 5.6

devishke-orange replied:
new license added

RajithaKumara commented on aws/assets/ssl.conf:
```diff
@@ -0,0 +1,52 @@
+<IfModule mod_ssl.c>
```
https://gist.github.com/RajithaKumara/87a69a196691083829ce4057dac47bdc#file-orangehrm-recommended-server-config-md
`X-frame-options` need to consider

RajithaKumara commented on aws/orangehrm:
```diff
@@ -0,0 +1,318 @@
+export ORANGEHRM_DB_VOLUME="orangehrm_orangehrm-db"
+
+# Update URL
+export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/master/aws/compose.yml"
```
```suggestion
export UPDATE_URL="https://raw.githubusercontent.com/orangehrm/os-cloud/main/aws/compose.yml"
```

devishke-orange replied:
changed to `main`

RajithaKumara commented on aws/tests/README.md:
````diff
@@ -0,0 +1,39 @@
+```
+5. Run script to setup tests
+```
+curl -o- https://raw.githubusercontent.com/devishke-orange/orangehrm-aws/main/tests/setup_tests.sh | bash
````
TODO

devishke-orange replied:
changed link

devishke-orange committed:
Added new License Text (#1)

* multiple changes
- added review feedback
- removed shebangs from sourced files
- packaging changes

* reverted license change and added GPLv3 license text

* changed license if condition

devishke-orange committed:
updated help guide link
